kahuna/public/js/components/gr-feature-switch-panel/gr-feature-switch-panel.module.css

```diff
@@ -0,0 +1,105 @@
+header {
+  display: flex;
+  flex-direction: row;
+  align-content: center;
+  border-bottom: 1px solid #565656;
+  padding: 10px;
+  padding-bottom: 4px;
+}
+
+table {
+  width: 400px;
+  margin-top: 0px;
+}
+
+table, th, td {
+  border: 1px solid #666;
+  border-collapse: collapse;
+}
+
+th, td {
+  padding: 2px 8px ;
+}
+
+th {
+  text-align: left;
+  padding-left: 6px;
+}
+
+
+.closeButton {
+  fill: rgb(204,204,204);
+  width: 20px;
+}
+
+.closeButton:hover {
+  fill: white;
+}
+
+.panel {
+  position: fixed;
+  top: 50px;
+  right: 0px;
+  width: 400px;
+  background-color: #333;
+  box-shadow: 2px 2px 10px rgba(0,0,0, 0.8);
+}
+
+.panelTitle {
+  display: inline-block;
+  flex-grow: 1;
+}
+
+.toggleButton {
+  padding: 4px 8px;
+  background-color: #444;
+  border: 1px solid #565656;
+}
+
+.toggleButtonAlt {
+  padding: 4px 8px;
+  background-color: #444;
+  border: 1px solid #565656;
+}
+
+.toggleButtonAlt:first-of-type {
+  border-top-left-radius: 4px;
+  border-bottom-left-radius: 4px;
+}
+
+.toggleButtonAlt:last-of-type {
+  border-top-right-radius: 4px;
+  border-bottom-right-radius: 4px;
+}
+
+.toggleButtonActive {
+  background-color: #666;
+  box-shadow: inset 0px 4px 0px -1px #00adee;
+}
+
+.toggleButton:last-of-type {
+  border-left: none;
+}
+
+.instructions {
+  padding: 4px;
+
+}
+
+.infoIcon {
+  display: inline;
+  fill: rgb(204,204,204);
+  width: 20px;
+}
+
+.infoPane {
+  border-bottom: 1px solid #565656;
+  width: 100%;
+  display: flex;
+  padding: 0px 8px;
```

Comment:
this infopane overflows for me, creating a horizontal scrollbar (because 100% width with padding). I think you could set `box-sizing: border-box` to fix it

Reply:
Good spot - now sorted